src/frontend/hooks/server/media.ts

```diff
@@ -1,52 +1,99 @@
 import {Observation} from '@mapeo/schema';
 import {BlobVariant} from '@mapeo/core/dist/types';
-import {useMutation, useQuery} from '@tanstack/react-query';
+import {useMutation, useQueries} from '@tanstack/react-query';
+import {SetRequired} from 'type-fest';
+import {URL} from 'react-native-url-polyfill';
```

Comment:
opted for the "convenient" usage of the polyfill here (https://github.com/charpeni/react-native-url-polyfill?tab=readme-ov-file#option-3-convenient--ponyfill). we could consider https://github.com/charpeni/react-native-url-polyfill?tab=readme-ov-file#option-1-simple but didn't think it was necessary to for now, given the very limited use case.